testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/OIDCClientRegistrationTest.java

```diff
@@ -356,6 +356,44 @@ public void testMtlsHoKTokenEnabled() throws Exception {
 
     }
 
+    @Test
+    public void testDPoPHoKTokenEnabled() throws Exception {
+        // create (no specification)
+        OIDCClientRepresentation clientRep = createRep();
+
+        OIDCClientRepresentation response = reg.oidc().create(clientRep);
+        Assert.assertEquals(Boolean.FALSE, response.getDpopBoundAccessTokens());
+        Assert.assertNotNull(response.getClientSecret());
+
+        // Test Keycloak representation
+        ClientRepresentation kcClient = getClient(response.getClientId());
+        OIDCAdvancedConfigWrapper config = OIDCAdvancedConfigWrapper.fromClientRepresentation(kcClient);
+        assertTrue(!config.isUseDPoP());
```

Comment:
Hint: It is slightly better to use `assertFalse(config.isUseDPoP());` . But no blocker for this PR IMO (rather a hint for the next time :-)

Reply:
Thank you for your suggestion. I will user `assertFalse` when I send PR in such cases in the future.